lib/fob.ex

```diff
@@ -187,4 +254,34 @@ defmodule Fob do
       %PageBreak{column: column, value: get_in(record, [Access.key(key)])}
     end)
   end
+
+  @doc since: "0.1.0"
+  @spec between_bounds(
+          Ecto.Queryable.t(),
+          [PageBreak.t()] | nil,
+          [PageBreak.t()] | nil
+        ) ::
+          Ecto.Query.t()
+  def between_bounds(queryable, start, stop)
+
+  def between_bounds(queryable, start, stop) do
+    query = Ecto.Queryable.to_query(queryable)
+    start = start |> PageBreak.add_query_info(query)
+    stop = stop |> PageBreak.add_query_info(query) |> reverse()
+
+    query
+    |> apply_keyset_comparison(start, :lenient)
+    |> apply_keyset_comparison(stop, :lenient)
+  end
```

Comment:
pretty straightforward and slim

the main point here is that we reverse the ordering of the page-breaks for `stop`. that reverses the ordering (`:asc -> :desc` and vice-versa etc.). that allows us to reuse the `apply_keyset_comparison/3` function

Reply:
nice move